packages/askui-nodejs/src/lib/ui-controller-args.ts

```diff
@@ -43,6 +43,7 @@ export interface UiControllerArgs {
   readonly logLevel?: LogLevels;
   readonly logFilePath?: string;
   readonly proxyAgents?: ProxyAgentArgs
+  readonly runtime?: 'desktop' | 'android'
```

Comment:
That was the '-r  android' we used, right? 

I guess for now we have some limitations here. Later I would prefer a '--device <device_name>' option.